### DISCUSSION TITLE
How Do I Clear IBM i Editor History?

### DISCUSSION THREAD

**Discussion by HerbertR** [codefori/vscode-ibmi#2123] [General]
I love that Code for IBM i/RPG Languages remembers where I was from day to day, so that when I re-open the source member, I can start where I left off. However, after completing the block of work and implementing it, I would like to clear out all of that cache, so that I start fresh the next time I work on any of the previously opened source members.

How do I clear out the cache that holds on to the location-in-the-source-members from the prior editing session?

**Comment by worksofliam** [comment]
If this is the quick pick menu you are using: scroll to the bottom of the list and there should be a clear button you can use to clear out all the entries.

**Reply by HerbertR** [reply]
I'm using the Object Browser to open source members. Basically, I open VS Code, connect to the dev IBM i, open a source member, do some work (or just browse it), log off. Next day, same basic work flow, and when I open the source member from Object Browser, it remembers where I was the last time that source member was opened.

(I'm new to VS Code/Code for IBM i, so I'm not very skilled with it yet. I actually don't know what the Quick Pick menu is)
